examples/demo_prerecorded.py

```diff
@@ -0,0 +1,54 @@
+import asyncio
+import os
+from deepgram import create_client
+from deepgram.deepgram_client import DeepgramClient
+from dotenv import load_dotenv
+load_dotenv()
+
+API_KEY = os.getenv('DG_API_KEY')
+AUDIO_FILE = "preamble.wav"
+AUDIO_URL = {
+    "url": "https://static.deepgram.com/examples/Bueller-Life-moves-pretty-fast.wav"}
```

Comment:
Style: do we use any code formatters? [Black](https://pypi.org/project/black/) is a common one for Python. It would be nice to run this through a code formatter all in one go, so we don't end up needing to make handfuls of formatting fixes which can clutter PR diffs later.

Reply:
+1! It would be very nice to use the Black formatter.

Reply:
I'm creating a issue to include this in the project. So we'll add it as part of the v3 release (but in another PR). https://github.com/deepgram/deepgram-python-sdk/issues/162